osbs/core.py

```diff
@@ -741,7 +741,7 @@ def adjust_attributes_on_object(self, collection, name, things, values, how):
         """
         url = self._build_url("%s/%s" % (collection, name))
         response = self._get(url)
-        logger.debug("before modification: %s", response.content)
+        logger.debug("before modification: %s", response.text)
```

Comment:
There is:
* the actual encoding of the response data
* the *declared* encoding of the response data in the headers
* the output encoding for logger.debug

This looks like it will fix a mismatch between the actual response encoding and the output encoding, but it won't help if the declared encoding is wrong -- whereas the `content` property will.

Don't we want `"%r", response.content` instead? That ought to cover all encoding mismatches.

Reply:
Sounds good. I will run a few tests in osbs-box and then squash the revert and the new commit.

Reply:
Done